ios/RNSScreenStack.mm

```diff
@@ -382,9 +382,13 @@ - (void)setModalViewControllers:(NSArray<UIViewController *> *)controllers
   [newControllers removeObjectsInArray:_presentedModals];
 
   // We need to find bottom-most view controller that should stay on the stack
-  // for the duration of transition. There are couple of scenarios:
+  // for the duration of transition.
+  // We will want to dismiss all
```

Comment:
...to dismiss all what? 😄

Reply:
+1, this is leftover from my side. I've decided to write that information in different place (somewhere below).

https://github.com/software-mansion/react-native-screens/pull/2113/commits/9e6f2f9f74f43c28b02f3a24d61616e1546cccc0